target/linux/ath79/image/mikrotik.mk

```diff
@@ -1,5 +1,16 @@
 include ./common-mikrotik.mk
 
+define Device/mikrotik_routerboard-493ah
+  $(Device/mikrotik)
```

Comment:
You can use (recently added) Device/mikrotik_nand for this include instead, and then drop IMAGE/sysupgrade.bin and DEVICE_PACKAGES below.